docs/platforms/react-native/manual-setup/expo/expo-sagp.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Sentry Android Gradle Plugin (AGP)
+description: "Learn how to configure the Sentry Android Gradle Plugin to automatically upload source maps and debug symbols for your Expo project."
+---
+
+<Alert>
+
+The **Sentry AGP** is still an experimental option of the Sentry React Native Expo plugin. If you encounter any issues please let us know via a GitHub ticket.
+
+</Alert>
+
+To enable the plugin modify the Expo application configuration and add the `enableAndroidGradlePlugin` option in the `@sentry/react-native/expo` plugin section.
+
+```javascript {tabTitle:app.json/app.config.json}
+{
+  "expo": {
+...
+    "plugins": [
+      [
+        "@sentry/react-native/expo",
+        {
+          "url": "https://sentry.io/",
+          "project": "project-slug",
+          "organization": "organization-slug",
+          "experimental_android": {
+            "enableAndroidGradlePlugin": true,
+          }
+        }
+      ],
+...
+  }
+}
+```
```

Comment:
The `{10-12}` highlights the code on the website. And `___PROJECT_SLUG___` and `___ORG_SLUG___` are replaced with interactive drop-down.

````suggestion
```javascript {tabTitle:app.json/app.config.json} {10-12}
{
  "expo": {
    "plugins": [
      [
        "@sentry/react-native/expo",
        {
          "url": "https://sentry.io/",
          "project": "___PROJECT_SLUG___",
          "organization": "___ORG_SLUG___"
          "experimental_android": {
            "enableAndroidGradlePlugin": true,
          }
        }
      ],
  }
}
```
````